<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceList</class>
 <widget class="QMainWindow" name="DeviceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="siteIdLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="siteIdPushButton">
        <property name="text">
         <string>siteId</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceSerialLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="deviceSerialPushButton">
        <property name="text">
         <string>deviceSerial</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
